app/views/submissions/submit_comment.erb

```diff
@@ -1,14 +1,18 @@
-<form accept-charset="UTF-8" action="/submissions/<%= submission.key %>/nitpick" method="POST">
+<% if current_user.guest? %>
+  <p> You're not logged in right now.Please login via <a href="/login">GitHub </a> to comment </p>
+<% else %>
+  <form accept-charset="UTF-8" action="/submissions/<%= submission.key %>/nitpick" method="POST">
   <%= erb :"submissions/comment", locals: { submission: submission } %>
 
-  <button type="submit" name="nitpick" class="pull-right btn btn-primary btn-lg">
-    Submit Comment
-  </button>
+    <button type="submit" name="nitpick" class="pull-right btn    btn-primary btn-lg">
```

Comment:
I don't know it is the editor. I tried it though. May be I get the class to next line.